server/src/pages/shot/view.js

```diff
@@ -350,7 +166,7 @@ class Frame extends React.Component {
   }
 
   onClickUploadFullPage(e) {
```

Comment:
At this point maybe we should remove more code related to old features that are no longer implemented? The product is firming up a lot and it's unlikely that we will implement new features by using old code, but we will implement them with new code instead.